nodes/widgets/locales/en-US/ui_text.html

```diff
@@ -13,6 +13,16 @@
             <li><b>Text Color:</b> Color picker to define the color to display the text in.</li>
         </ul>
     </p>
+    <h3>HTML Injection</h3>
+    <p>
+        The text field can also be used to inject HTML into the user interface.
```

Comment:
```suggestion
        The text field can also be used to add HTML into the user interface.
```

inject has connotations - maybe say "add" instead?